### PR TITLE
Use singular for consistency

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Use singular for consistency.

### CODE REVIEW

File: tests/si_aggregator_test.py [hunk 1]
```diff
@@ -78,5 +78,7 @@ def test_create_si_aggregated_json(self) -> None:
         with open(self._SI_AGGREGATOR_REPORTS_FILE) as json_file:
             aggregated_json_test = json.load(json_file)
         keys_tests = sorted([k for k in aggregated_json_test.keys()])
+        print(keys)
```
Comment (saisankargochhayat) [inline tests/si_aggregator_test.py]: Are these prints intentional?

Comment (pacospace) [reply tests/si_aggregator_test.py]: Nice catch!! Removed them in af11332db026a6ed160f06ec7597d6f57a5568be

**Review by saisankargochhayat — APPROVED** [review]
LGTM

**Review by KPostOffice — APPROVED** [review]

**Pull request merged**